docs/features/ems_reporting.md

```diff
@@ -41,24 +42,24 @@ A scheduled Lambda task will run nightly that collects distribution events and b
 
 Information about requesting EMS account can be found on EMS website.  Here are basic steps in order to submit reports to EMS.
 
-1. Get a provider account on the EMS file server and obtain access to their UAT or OPS environment.
+### 1) Get a provider account on the EMS file server and obtain access to their UAT or OPS environment
```

Comment:
I assume this was to fix linting errors since there is multi-line content between each ordered list item. You can also fix it by leaving the list item as `1.` and double indenting the content between each list item, which I think I like better. Better yet, the two points below this numbered item should be an unordered list for readability

Reply:
It led to rendering weirdness, at least in the preview.   I also like that in theory.... you can _probably_ link to this, although I didn't think through that too deeply.

Reply:
That said I'm not opposed to changing it.

Reply:
I guess I prefer a regular ordered list as opposed to making them headers, but I don't feel super strongly. Your call